content/en/community/hackathons/usoc23/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Unikraft Summer of Code 2023"
+date: 2023-04-12T01:10:00+02:00
+draft: false
+collapsible: true
+description: Unikraft Summer of Code 2023, July 3 - 15, 2023
+image: /assets/imgs/usoc23.png
+---
+
+## Unikraft Summer of Code 2023
+
+Unikraft Summer of Code 2023 (*USoC'23*) is a **free and virtual** unikernel and library operating systems workshop held by members of the Unikraft community.
+
+From zero-to-hero the two week event holds a number of starter tutorials, workshops and lessons on how to get build Unikraft unikernels for production systems.
+
+The workshop will be hands-on and will take place for 10 days, between August 29 and September 9, 2022, 4pm-7pm CEST, in English, virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
+Topics include building unikernels, benchmarking, debugging, porting applications, virtualization and platform specifics.
+The workshop will take place for two weeks (July 03 - July 16, 4pm-7pm CEST) virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
+There will be six 3-hour sessions, followed by a full day hackathon which will be held live in `TBA` at [University POLITEHNICA of Bucharest](https://upb.ro), with the possibility to also join online via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
+
+### Registration
+
+If you're eager to learn more about operating systems and unikernels, to work on practical open source tasks, to expand your knowledge of low-level topics, you'll want to be part of USoC'23.
+To take part in the event, please fill this [registration form](https://forms.gle/ZUVLnkb6YNWhr9Cp8).
+You will need to complete a set of challenges that will get you accustomed to the environment you will be using during the sessions.
+
+It's recommended you check these prerequisites before taking part in USoC'23:
+
+* fair knowledge of Linux command-line interface
```

Comment:
```suggestion
* good knowledge of Linux command-line interface
```

Reply:
I would leave this to `fair`, Linux cli is not that strong of an item within Unikraft.